packages/client/ui/react-ui/src/icons/base64Icons.ts

```diff
@@ -0,0 +1,8 @@
+const metamaskIcon =
```

Comment:
I decided to try out using base64 images. I feel it's appropriate for smaller bundle sizes and easier to use for this package. 
(** note; I compressed the original image for these by upwards of ~90%)